### DISCUSSION TITLE
Turn Muted Mod into a Metronome Mod with optional muting

### DISCUSSION THREAD

**Discussion by InvisibleSymbol** [ppy/osu#14056] [Ideas]
It feels weird that I can toggle the metronome, but can't control if it mutes the music or not. I think it would be nice to have the option of using the metronome as a kind of support with the music. If one wants complete silence they should just adjust their volume for that imho.
My suggestion is to turn the muted mod into a metronome mod and give the option of muting the music if one wants to, compared to always muting the music

**Comment by frenzibyte** [comment]
The mod may change from completely silencing the beatmap track to instead gradually soften as the user gets closer to the defined combo number (see [discord discussion](https://ptb.discord.com/channels/188630481301012481/188630652340404224/870334210777419796)), so it will become more than just "set music volume to 0%".

And either way, I'm not quite sure just how good having a mod that pretty much adds a metronome in the background without any other effects. It doesn't sound like it would be fun in my opinion, would it?

**Reply by InvisibleSymbol** [reply]
> And either way, I'm not quite sure just how good having a mod that pretty much adds a metronome in the background without any other effects. It doesn't sound like it would be fun, in my opinion, would it?

Having a metronome with music sound more fun than having no music at all :P

Turning the mod into primarily a metronome one provides the user with more options than with the current setup without adding any complexity:
- Want music with an extra metronome? Enable the metronome mod (currently impossible)
- Want no music but still a metronome? Enable the mute option
- Want no music or metronome? Turn down music volume

The "fade to silence on combo" idea could also be applied to: both the metronome and music; or only the metronome, based on what the user selects. Fading only music would not be possible using this setup, but I would argue that at that point the fading part should be separated into its own mod.

**Reply by peppy** [reply]
want unlimited possibilities and toggles for every little thing with no opinionated fun anywhere? toggle everything everywhere ever. let's split metronome mod out into four mods - one for each beat in a bar while we're at it.

disagree with splitting. settings can be added to be mod as i proposed, which will handle most cases.
